### DISCUSSION TITLE
v19.17.2-3.0.1 - YTLitePlus

### DISCUSSION THREAD

**Discussion by Balackburn** [YTLitePlus/YTLitePlus#141] [Announcements]
# ⚠️ Must Read ⚠️
- The .ipa file is no longer shared on Github due to a [Google DMCA notice](https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/) 
- You need to build it yourself. See [Wiki-Building](https://github.com/Balackburn/YTLitePlus/wiki/Building)

# YTLitePlus Release Information

Current YouTube IPA: `19.17.2`  
Current YTLite Version: `3.0.1`  
Current App Compatibility: `iOS/iPadOS 14.0` or later

**RELEASE 19.17.2-3.0.1:** (2024-05-05)

- Updated Youtube from `19.16.3` to `19.17.2`
- Updated submodules
- Removal of custom icons (because they weren't functional)
  
### Issues in this Release:
- LowContrastMode does not work *(Compatible only with YouTube v17.33.2-v17.38.10)*
- VersionSpoofer (Lite) doesn't work with YTLite *(as the version is already spoofed by YTLite to fix YTClassicVideoQuality)*
- Hiding Shorts creates an empty space instead #55

<hr /><em>This discussion was created from the release <a href='https://github.com/YTLitePlus/YTLitePlus/releases/tag/v19.17.2-3.0.1'>v19.17.2-3.0.1 - YTLitePlus</a>.</em>

**Comment by Valdularo** [comment]


**Reply by Avakining** [reply]
I think this is #116?

**Reply by Balackburn** [reply]
> Shorts isn't disabled btw.

Look at issues first instead of creating a discussion #116

**Comment by TV365-VN** [comment]
Original version V.19.18.2 is available

**Comment by dragonslid** [comment]
where can i find the direct link to the ipa to build

**Reply by Balackburn** [reply]
> where can i find the direct link to the ipa to build

https://armconverter.com/decryptedappstore/us
https://decrypt.day/
https://anyipa.me/

Or decrypt the .ipa from your jailbroken iOS device

Upload it to litterbox.catbox.moe and copy the link

**Reply by shahin365** [reply]
Can I use the latest decrypted YouTube .ipa version **19.18.2** that I downloaded today (instead of the 19.17.2 mentioned here)?

**Reply by Balackburn** [reply]
> Can I use the latest decrypted YouTube .ipa version **19.18.2** that I downloaded today (instead of the 19.17.2 mentioned here)?

Yes

**Reply by shahin365** [reply]
successfully forked. thank you for your help.

**Comment by TV365-VN** [comment]
Bạn có thể tạo ra Netflix Premium không

**Comment by Balackburn** [comment]
> Bạn có thể tạo ra Netflix Premium không

No I can't
